recipe/meta.yaml

```
@@ -27,38 +28,57 @@ outputs:
         - python
         - packaging
         - pandas >=1.4.0
+        - shapely >=2.0.0
+        - numpy >=1.22
+        - pyogrio >=0.7.2
         - pyproj >=3.3.0
-        - shapely >=1.8.0
-        - fiona >=1.8.21
+      run_constrained:
+        # geodatabase access
+        - psycopg2-binary>=2.8.0
+        - psycopg-binary>=3.1.0
+        - SQLAlchemy>=1.3
```

```suggestion
        - psycopg-binary >=3.1.0
        - sqlalchemy >=1.3
```
Whitespaces and normalizing the name